tests/fast-integration/skr-svcat-migration-test/skr-svcat-migration-test.js

```diff
@@ -28,17 +31,20 @@ describe("SKR SVCAT migration test", function() {
   const suffix = genRandom(4);
   const appName = `app-${suffix}`;
   const runtimeName = `kyma-${suffix}`;
-  const runtimeID = uuid.v4();
+  const instanceID = uuid.v4();
```

Comment:
thanks for fixing this as well! I now have much clearer understanding to distinguish between instance_id and runtime_id so hopefully won't make many of these in the future :)